prow/kube/config.go

```diff
@@ -134,6 +134,9 @@ func LoadClusterConfigs(opts *Options) (map[string]rest.Config, error) {
 		}
 	}
 
+	if opts.skipMergeConfigs {
```

Comment:
Sorry, I don't understand. What expectations could someone who sets this possible have? That they get a random kubeconfig out of the possible options that are configured? That doesn't seem useful.

Reply:
Probably the function name `mergeConfigs` is misleading.

https://github.com/kubernetes/test-infra/blob/8b97dae4b0d8e5f7aa75dc911ea22cb00ad5e0bd/prow/kube/config.go#L51-L70

It adds the keys "" (the empty string) and "default" into the map, besides the loaded ones.

So with this option, it will return the maps without adding those two keys.

Reply:
How about simply "NoInClusterConfig"?

Reply:
Updated.